### PR TITLE
add zip and unzip

### PR DESCRIPTION
**Pull request by msarahan**
See https://github.com/conda-forge/tensorflow-feedstock/pull/12#discussion_r76100563

CC @jakirkham 

Alternatively, we could just yum install these by default.

### CODE REVIEW

File: recipes/unzip/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  name: unzip
+  version: {{ version }}
+
+source:
+  url: ftp://ftp.info-zip.org/pub/infozip/src/unzip60.tgz
+  sha256: 036d96991646d0449ed0aa952e4fbe21b476ce994abc276e49d30e686708bd37
+
+build:
+  number: 0
+  skip: True  # [not linux]
```
Comment (jakirkham) [inline recipes/unzip/meta.yaml]: Maybe do Unix. That way we can use them on OS X if we want.

File: recipes/zip/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  name: zip
+  version: {{ version }}
+
+source:
+  url: ftp://ftp.info-zip.org/pub/infozip/src/zip30.tgz
+  sha256: f0e8bb1f9b7eb0b01285495a2699df3a4b766784c1765a8f1aeedf63c0806369
+
+build:
+  number: 0
+  skip: True  # [not linux]
```
Comment (jakirkham) [inline recipes/zip/meta.yaml]: Ditto.

File: recipes/unzip/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{% set version="6.00" %}
+
+package:
+  name: unzip
+  version: {{ version }}
+
+source:
+  url: ftp://ftp.info-zip.org/pub/infozip/src/unzip60.tgz
```
Comment (jakirkham) [inline recipes/unzip/meta.yaml]: Probably need to template this URL somehow to get the version out.

File: recipes/zip/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{% set version="3.00" %}
+
+package:
+  name: zip
+  version: {{ version }}
+
+source:
+  url: ftp://ftp.info-zip.org/pub/infozip/src/zip30.tgz
```
Comment (jakirkham) [inline recipes/zip/meta.yaml]: Also version templating.

File: recipes/unzip/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
+{% set version="6.00" %}
```
Comment (jakirkham) [inline recipes/unzip/meta.yaml]: It looks like it is just `6.0`.

File: recipes/zip/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,28 @@
+{% set version="3.00" %}
```
Comment (jakirkham) [inline recipes/zip/meta.yaml]: Looks like it is just 3.0.

**Comment by jakirkham** [comment]
I'm in favor of this. Some notes.

The versioning of these is weird as they sometimes use `0` and sometimes use `00`. They do sometimes include two digits in the minor version. Not sure what we want to do there.

Might as well build them on Mac too. Can't really see any reason not too.

The URLs should be templated with the version somehow.

**Comment by jerowe** [comment]
@jakirham, thanks so much! You have been so helpful! Hopefully this is the beginning of a beautiful relationship. ;-)

You wouldn't happen to be involved in conda-env, would you? Because I've got some pending PRs there too...

**Comment by jakirkham** [comment]
Nope, sorry, I don't work for Continuum so can't help you there. 😄

**Comment by jerowe** [comment]
How should this work, exactly? I can't fork @msarahan 's fork, because I've already forked the main repo, but it would be nice to show he started the recipe.

**Comment by msarahan** [comment]
@jerowe you can add my repo as a remote:

    git remote add msarahan https://github.com/msarahan/staged-recipes

Then fetch

    git fetch msarahan

then checkout the branch

    git checkout zip

finally, push up your own branch to github, and either create a new PR against conda-forge/staged-recipes, or PR against my repo, and I'll be happy to merge your changes as part of this PR.

**Comment by jerowe** [comment]
@msarahan , thanks! I will get started.

**Comment by jakirkham** [comment]
Also one can checkout an arbitrary PR by doing something like this. Where `<PR_NUM>` is the number of the PR one wants.

```
$ git fetch https://github.com/conda-forge/staged-recipes refs/pull/<PR_NUM>/head:pr_<PR_NUM>
```

This will create a branch `pr_<PR_NUM>` that can be checked out and starts on the latest commit of the PR.

**Comment by jerowe** [comment]
Is this bad?

https://travis-ci.org/jerowe/staged-recipes/builds/162268806#L558

**Comment by jerowe** [comment]
@msarahan , I used your suggestion and generated a pull request against your repo. It built at least on Travis, but I don't have Circle or AppVeyor set up on my github account.

**Comment by jakirkham** [comment]
> Is this bad?

> https://travis-ci.org/jerowe/staged-recipes/builds/162268806#L558

Nope, that's fine. libSystem is like Mac's equivalent to GLIBC on Linux. Basically, it provides a base level C API that is used to perform system operations. It's not something that we would want to package.

**Comment by jakirkham** [comment]
Is it alright if we close this so as to continue the discussion in PR ( https://github.com/conda-forge/staged-recipes/pull/1671 ), @msarahan?

**Comment by msarahan** [comment]
Sure - closing.  Thanks @jerowe for taking over.

**Pull request closed without merging**